BMM.UI.iOS/BMM.UI.iOS.csproj

```diff
@@ -138,6 +138,12 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+  <ItemGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <NativeReference Include="$(HOME)\Library\Application Support\Reveal\RevealServer\RevealServer.xcframework">
```

Comment:
Interesting. Do you have a license for Reveal? Maybe you can show me how it works.